client/src/pages/Login.js

```diff
@@ -110,6 +164,14 @@ const useStyles = makeStyles(theme => ({
     },
     TextFields: {
         width: '100%',
+        '& label.Mui-focused': {
+            color: theme.primary,
+        },
+        '& .MuiOutlinedInput-root': {
+            '&.Mui-focused fieldset': {
+                borderColor: theme.primary,
+            },
+        },
```

Comment:
This is going to get repetitive to copy and paste in all the components that we have. I have two ideas to make this more reusable:
- we use theme overrides for components in the theme.js file 
- we create reusable specific components like below:


```
const StyledPaper = ({children, ...otherProps}) => {
      const classes = useStyles(); 
      return <Paper className={classes.paper} {...otherProps}>{children}</Paper>
}
```